rules/python/shared/common/user_input.yml

```
@@ -0,0 +1,19 @@
+type: shared
+languages:
+  - python
+imports:
+  - python_shared_django_user_input
+  - python_shared_lang_user_input
+patterns:
+  - pattern: $<PYTHON_SHARED_COMMON_USER_INPUT>
+    filters:
+      - either:
+          - variable: PYTHON_SHARED_COMMON_USER_INPUT
+            detection: python_shared_django_user_input
+            scope: cursor
```

Doesn't really matter, but you can do this which is slightly better performance-wise
```suggestion
            scope: cursor_strict
```